mkinitcpio-install.sh

```diff
@@ -224,7 +232,13 @@ add_systemd_unit_X() {
                 ;; 
         esac
 
-    done <"$unit"
+    done < <(systemctl cat "${unit##*/}")
+
+    # add drop-in files
+    local dropin
+    for dropin in $(systemctl show  --property=DropInPaths --value "${unit##*/}"); do
+      add_file "$dropin"
```

Comment:
Maybe instead of adding each drop-in file, we could opt for adding composite files based on `systemd cat <service>`?

Reply:
there you go